scipy/optimize/_differentialevolution.py

```diff
@@ -106,12 +106,18 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         If True (default), then `scipy.optimize.minimize` with the `L-BFGS-B`
         method is used to polish the best population member at the end, which
         can improve the minimization slightly.
-    init : string, optional
-        Specify how the population initialization is performed. Should be
+    init : string or array-like, optional
+        Specify which type of population initialization is performed. Should be
         one of:
 
             - 'latinhypercube'
             - 'random'
+            - array specifying subset of the initial population. The array can
+              be of shape ``(len(x),)`` or ``(M, len(x))``, where len(x) is the
+              number of parameters. A total of ``min(M, popsize * len(x))``
```

Comment:
It's also not clear to me why you have two options here. What does ``len(x)`` achieve?

EDIT: this is just a special case right, with ``M=1``? And such a small init size is not recommended? If I understood that correctly, then just removing the 1-D ``len(x)`` option would be good.